### ISSUE TITLE
Transaction list: remove infinite loading

### ISSUE THREAD

**Issue by edd** [vegaprotocol/frontend-monorepo#4070]
## The Chore

Inifinite loading works in some places, but it is not a good system for looking through history for transactions. We should remove it and switch to a manually paged list of transactions

## Tasks
- [ ] This should change all TX lists - the one on the party page and the one on the transactions page
- [x] Remove infinite loading wrapper
- [ ] Remove react-window wrapper around table
- [ ] Add more standard pagination controls. As a starting point, let's 
   - [ ] This can be iterated on, but must have:
   - [ ] Previous page button, which should be disabled on page 1
   - [ ] Current page number
   - [ ] Next page button
- [x] Page size should be 25
    - [ ] A follow up ticket can make this configurable with a dropdown

## Notes
- This doesn't need to change the layout. That's a separate ticket.
- It can change the layout if it's better done all at once.

**Comment by edd** [comment]
Sort of semi blocked by https://github.com/vegaprotocol/vega/issues/8545 but I'll implement a hack
